packages/api/endpoints/granules.js

```diff
@@ -260,8 +264,36 @@ const patchGranule = async (req, res) => {
   return _returnPatchGranuleStatus(isNewRecord, apiGranule, res);
 };
 
-function put(req, res) {
-  return res.boom.badRequest('put method not implemented');
+/**
+ * Replace a single granule
+ * @param {Object} req - express request object
+ * @param {Object} res - express response object
+ * @returns {Promise<Object>} the promise of express response object
+ */
+async function put(req, res) {
```

Comment:
@Jkovarik it looks like `put` doesn't support actions. Is that right and is PATCH the endpoint that we'll use to (e.g.) applyWorkflow? It seems like that could be right but I wanted to verify because that would be a breaking change I think as PUT currently supports actions.

Reply:
@npauzenga  yes, that's intended.  I think that's correct (insofar as having actions in a PUT/PATCH endpoint *at all* is correct) as those actions more correctly align with PATCH (modification of but not necessarily complete replacement of the record).

I'm open to other thoughts on the approach if you feel strongly.